obspy/taup/tau.py

```diff
@@ -663,3 +713,88 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(source_depth, min_degree=0, max_degree=360, npoints=1000,
+                    phases=['P', 'S'], model='iasp91',
+                    ax=None, show=True):
+        """
+        Method to plot the travel times of arrivals class, if any have
+        been calculated.
+
+        :param source_depth: Source depth in kilometers.
+        :type source_depth: float
+        :param min_degree: minimum distance from the source (in degrees) to a
+            plot travel times Defaults to ``0``.
+        :type min_degree: float
+        :param max_degree: maximum distance from the source (in degrees) to
+            plot travel times. Defaults to ``360``.
+        :type max_degree: float
+        :param npoints: Number of points to plot. Defaults to ``1000``.
+        :type npoints: int
+        :param phases: List of phase names which should be used within the
+            plot. Defaults to ``['P', 'S']``.
+        :type phases: list of str, optional
+        :param model: string containing the model to use. Defaults to 'iasp91'.
+        :type model: str
+        :param ax: Axes to plot to. If not given, a new figure with an axes
+            will be created.
+        :type ax: :class:`matplotlib.axes.Axes`
+        :param show: Show the plot.
+        :type show: bool
+
+        :returns: The (possibly created) axes instance.
+        :rtype: :class:`matplotlib.axes.Axes`
+
+        .. rubric:: Example
+
+        >>> from obspy.taup import traveltime_plot
+        >>> traveltime_plot(source_depth=120, min_degree=0, max_degree=50,
+        ...          phases=['P', 'S', 'PP'], model='iasp91', npoints=1000)
```

Comment:
Need to add the return value to the doctest, see https://ci.appveyor.com/project/obspy/obspy/build/1.0.5123-master/job/gjp7dcd9yjp7vq5u#L719:

```

        >>> traveltime_plot(source_depth=120, min_degree=0, max_degree=50,
        ...          phases=['P', 'S', 'PP'], model='iasp91',
        ...          npoints=1000)  # doctest: +ELLIPSIS
        <matplotlib.axes._subplots.AxesSubplot object at 0x...>

Reply:
The return value *is* in the doc string, but should it be at the very end of the string? There was an example after the return value. I am changing the order, but is that enough?

Reply:
> The return value is in the doc string

Sorry, I don't understand what you're saying. `traveltime_plot` is called without assigning it's return value and so the interactive prompt will print it's representation after the call and the result of this interactive print is missing in your doctest example. Just change it like I proposed and it should work.

> that's the same error I got when creating the test image!

That's strange, because in the unit test no return values are automatically printed/asserted, only explicit unit test assertions. But of course, if you run `$ obspy-runtests taup --keep-images` then doctests are executed as well (as opposed to running `$ OBSPY_KEEP_IMAGES= python taup/tests/test_plotting.py` which really only runs the unit tests in that file).